test/Renci.SshNet.Tests/Classes/ShellStreamTest.cs

```diff
@@ -123,6 +123,18 @@ public void Write_AfterDispose_ThrowsObjectDisposedException()
             Assert.ThrowsException<ObjectDisposedException>(() => shellStream.Write(bytes, 0, bytes.Length));
         }
 
+        [TestMethod]
+        public void SendWindowChangeRequest_ThrowsObjectDisposedException()
```

Comment:
Could you also add a simple test which verifies the call to `ChannelSession.SendWindowChangeRequest`?